vega_sim/scenario/ideal_market_maker/scenario.py

```diff
@@ -188,14 +192,26 @@ def set_up_background_market(
             commitamount=self.lp_commitamount,
         )
 
+        info_trader = InformedTrader(
+            wallet_name = INFORMED_WALLET.name,
+            wallet_pass = INFORMED_WALLET.passphrase,
+            price_process = price_process,
+            market_name = market_name,
+            asset_name = asset_name,
+            initial_asset_mint = self.initial_asset_mint,
+            proportion_taken = self.proportion_taken,
+            tag = str(tag),
+        )
+
         env = MarketEnvironment(
             base_agents=[
                 market_maker,
                 tradingbot,
                 randomtrader,
                 auctionpass1,
                 auctionpass2,
-                # liquidityprovider,
+                info_trader,
+                liquidityprovider,
```

Comment:
Looks like liquidity provider was commented out here before, is uncommenting deliberate?